modules/nested-accordion/widgets/nested-accordion.php

```diff
@@ -799,6 +828,28 @@ protected function render() {
 			<?php echo $items_title_html; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
 		</div>
 		<?php
+
+		if ( $settings['faq_schema'] ) {
+			$json = [
+				'@context' => 'https://schema.org',
+				'@type' => 'FAQPage',
+				'mainEntity' => [],
+			];
+
+			foreach ( $faq_schema as $name => $text ) {
+				$json['mainEntity'][] = [
+					'@type' => 'Question',
+					'name' => wp_strip_all_tags( $name ),
+					'acceptedAnswer' => [
+						'@type' => 'Answer',
+						'text' => wp_strip_all_tags( $text ),
+					],
+				];
+			}
+		}
+		?>
+		<script type="application/ld+json"><?php echo wp_json_encode( $json ?? [] ); ?></script>
```

Comment:
The `<script>` tag should be inside the if block?

Reply:
Agree